src/outpack/sandbox.py

```diff
@@ -0,0 +1,79 @@
+import os
+import pickle
+import subprocess
+import sys
+
+from tblib import pickling_support
+
+from outpack.util import openable_temporary_file
+
+
+def run_in_sandbox(target, args=(), cwd=None, syspath=None):
```

Comment:
We use `callr` for this sort of manoeuvre in R - does python not have something we can pull in to do this for us?  This way is fine though!

Reply:
All I could find is the multiprocessing module, but it's a bit inconvenient to use and puts a burden on the caller to setup their script in the right way.